Tasks/AzurePowerShellV4/package.json

```diff
@@ -4,11 +4,14 @@
         "name": "Microsoft Corporation"
     },
     "dependencies": {
-        "@types/mocha": "2.2.48",
-        "@types/node": "6.0.68",
+        "@types/mocha": "^5.2.7",
+        "@types/node": "^10.17.0",
         "@types/q": "1.0.7",
-        "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.3",
-        "azure-pipelines-task-lib": "2.8.0",
-        "utility-common-v2": "file:../../_build/Tasks/Common/utility-common-v2-2.0.0.tgz"
+        "azure-pipelines-task-lib": "3.0.6-preview.0",
```

Comment:
Could you please specify task-lib version as '^3.0.6-preview.0' as it specified in azure-pipelines-tasks-utility-common package? There is an issue with task-lib when different versions of this library are specified for task and used packages, so this could prevent it for future task-lib updates.

Reply:
done